ui/src/shared/components/FancyScrollbar.js

```diff
@@ -33,10 +33,10 @@ class FancyScrollbox extends Component {
 
 const {bool, node, string} = PropTypes
 
-FancyScrollbox.propTypes = {
+FancyScrollbar.propTypes = {
   children: node.isRequired,
   className: string,
   autoHide: bool,
 }
 
-export default FancyScrollbox
\ No newline at end of file
+export default FancyScrollbar
```

Comment:
looks like we might want a carriage return at the end of this -- is webpack yelling at you for this?